core/controllers/admin.py

```diff
@@ -203,6 +208,26 @@ def _reload_collection(self, collection_id):
                 feconf.SYSTEM_COMMITTER_ID, unicode(collection_id))
         else:
             raise Exception('Cannot reload a collection in production.')
+    def _reload_dummy_exploration(self, dummy_exp_count):
```

Comment:
I think this PR should have tests for the new code that it adds to the codebase.

Re adding tests for "reload demo explorations", that would be nice to do so, but it's optional (since that functionality isn't related to this PR). So it might be best to do those and other tests in a separate PR, if you like.

Thanks!